style.css

```diff
@@ -85,13 +86,49 @@ ul {
   position: relative;
   background-color: #eaeaea;
   overflow: auto;
+  min-width: 150px;
 }
 
 
 /* add item page, the container holding the form section */
 .form_container {
-    height: 400px;
-    width: 100%;
-    margin-bottom: 50px; 
+    height: 750px;
+    width: 60%;
```

Comment:
Width 60% is going to get difficult to read on small screens. Consider adding a min-width entry.

Reply:
added this in thanks